config/aic.php

```diff
@@ -42,8 +42,12 @@
 
     // Branding
     'branding' => [
-        'digital_publications' => [
-            'colors' => [
+        'colors' => [
+            'general' => [
+                '#1E3F49',
+                '#282829',
```

Comment:
Alexis would like this to be the defacto background option for custom banners across the site for now.